salsa-2022-tests/tests/compile_fail.rs

```diff
@@ -1,3 +1,4 @@
+#[rustversion::stable]
```

Comment:
Add this to make `compile_fail' only run on stable.
Is it ok?

Reply:
seems ok for now